modules/administration-guide/partials/proc_applying-the-bitbucket-application-link-secret.adoc

```diff
@@ -0,0 +1,63 @@
+:_content-type: PROCEDURE
+:description: Applying the Bitbucket application link Secret
+:keywords: applying-bitbucket, apply-the-bitbucket, apply-bitbucket, apply-secret, applying-secret, apply-a-secret, applying-a-secret, bitbucket, bitbucket-application, bitbucket-app, bitbucket-application-link, bitbucket-app-link, bitbucket-secret, secret
+:navtitle: Applying the Bitbucket application link Secret
+// :page-aliases:
+
+[id="applying-the-bitbucket-application-link-secret_{context}"]
+= Applying the Bitbucket application link Secret
+
+Prepare and apply the Bitbucket application link Secret as follows:
```

Comment:
I think for the abstract, you don't need the colon to precede the prereqs and procedure. An abstract by nature sets these up. 

So simply, `Prepare and apply the Bitbucket application link Secret.`

Reply:
Thank you, Jake. I'll do it as you suggested.